src/components/Filters.jsx

```diff
@@ -138,6 +131,10 @@ export const Filters = () => {
           value={price}
           onChange={priceHandler}
         />
+        <div className="flex-center flex-col">
+          Showing products under
+          <div style={{ fontWeight: "bold" }}>{price}</div>
```

Comment:
if you are already using utility classes then such inline styles can be avoided and used utilities instead!

Reply:
I do not have utility classes for font weights in my component library. So, used inline styles here.